temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsImpl.java

```diff
@@ -151,6 +164,23 @@ public WorkflowServiceStubsImpl(
     log.info(String.format("Created GRPC client for channel: %s", channel));
   }
 
+  private ScheduledExecutorService startConnectionBackoffResetter(long backoffResetFrequency) {
```

Comment:
Change parameter type to Duration and use milliseconds when initializing executor.

Reply:
Done